migrations/1585388899167-add_default_file_storage_provider.js

```diff
@@ -0,0 +1,76 @@
+const mongoose = require('mongoose');
+const CryptoJS = require('crypto-js');
+const { Configuration } = require('@schul-cloud/commons');
+
+const { info, error } = require('../src/logger');
+const { schoolModel: School } = require('../src/services/school/model');
+
+const { connect, close } = require('../src/utils/database');
+
+const StorageProvider = mongoose.model('storageProviderMigration', new mongoose.Schema({
+	type: { type: String, enum: ['S3'], required: true },
+	isShared: { type: Boolean },
+	accessKeyId: { type: String, required: true },
+	secretAccessKey: { type: String, required: true },
+	endpointUrl: { type: String, required: true },
+	region: { type: String },
+	maxBuckets: { type: Number, required: true },
+	schools: [{ type: mongoose.Schema.Types.ObjectId, ref: 'school' }],
+}, {
+	timestamps: true,
+}), 'storageproviders');
+
+module.exports = {
+	up: async function up() {
+		await connect();
+		const S3_KEY = Configuration.get('S3_KEY');
+
+		if (!process.env.AWS_SECRET_ACCESS_KEY) {
+			info('No AWS config found. Migration will be successful but won\'t change anything!');
+		} else {
+			if (!S3_KEY) {
+				error('You need to set process.env.S3_KEY to encrypt the old key!');
+			}
+
+			let schools = await School.find({})
+				.select(['_id'])
+				.lean({ virtuals: true })
+				.exec();
+			schools = schools.map((s) => s._id);
```

Comment:
```suggestion
			schools = schools.map((s) => s.id);
```
why not taking the string id property?